src/test/java/com/foundation/trello/hook/BoardHook.java

```diff
@@ -15,29 +15,46 @@
  * @version 0.0.1
  */
 public final class BoardHook {
-    private Board board;
+    private Context context;
     private RequestManagerAbstract requestManager;
 
     /**
      * This method constructor initializes the variables.
      *
-     * @param context initialize board and context attributes.
+     * @param context initializes context attribute.
      */
     public BoardHook(Context context) {
-        this.board = context.getBoard();
+        this.context = context;
     }
 
     /**
      * Makes a request for delete a Board by id.
      */
     @After("@delete-board")
     public void afterScenario() {
-        String endPoint = "/boards/".concat(board.getId());
+        String endPoint = "/boards/".concat(context.getId());
         String method = "delete";
         requestManager = FactoryRequest.getRequest(method);
         requestManager.setMethod(method);
         requestManager.setEndPoint(endPoint);
         Response response = requestManager.makeRequest();
         Log.getInstance().getLog().info(response);
+
+    }
+    /**
```

Comment:
please, add a space between the comment and the brace.

Reply:
Okay, that mistake was fixed